performancetest/src/main/java/org/corfudb/performancetest/CorfuTablePerformanceTest.java

```diff
@@ -0,0 +1,193 @@
+package org.corfudb.performancetest;
+
+import com.google.common.reflect.TypeToken;
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.runtime.collections.CorfuTable;
+import org.junit.Test;
+import java.io.IOException;
+import java.util.*;
+
+/**
+ * CorfuTablePerformanceTest is the subclass of PerformanceTest.
+ * This class is used for CorfuTable performance test, mainly for put and get operations.
+ * @author Lin Dong
+ */
+@Slf4j
+public class CorfuTablePerformanceTest extends PerformanceTest{
+    /**
+     * Seed used by Random Object.
+     */
+    private final long seed;
+    /**
+     * Boundary of the integer generated randomly.
+     */
+    private final int randomBoundary;
+    /**
+     * The time each CorfuTable performance test runs.
+     */
+    private final long time;
+    /**
+     * The max number of keys can generated by KeyValueManager.
+     */
+    private final int keyNum;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'keyNum' could be replaced by a local variable.](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651454)